nsv13/code/modules/mob/living/carbon/human/nsv_human_helpers.dm

```diff
@@ -0,0 +1,25 @@
+/**
+ * # `species_examine_font()`
+ *
+ * This gets a humanoid's special examine font, which is used to color their species name during examine / health analyzing.
+ * The first of these that applies is returned.
+ * Returns:
+ * * Metallic font if robotic
+ * * Cyan if a toxinlover
+ * * Yellow-ish if an Ethereal
+ * * Purple if plasmaperson
+ * * Rock / Brownish if a golem
+ * * Green if none of the others apply (aka, generic organic)
+*/
+/mob/living/carbon/human/proc/species_examine_font()
+	if((MOB_ROBOTIC in mob_biotypes))
+		return "sc_robotic"
+	if(HAS_TRAIT(src, TRAIT_TOXINLOVER))
```

Comment:
It is intentional; The colors are dedicated towards special behaviours more than individual species (which is why all of the ones that behave 'normally' are just green).

That is also the reason it is arranged in the chain of special behavior -> specific species checks (or well, special behavior that is in the code of a specific species instead of based on a trait or traits.

What you highlight is effectively what it's for, other species that would be robotic or had toxinlover as one of their traits.